databuilder/dummy_data/query_info.py

```diff
@@ -118,6 +119,13 @@ def from_variable_definitions(cls, variable_definitions):
         # For every column used in the query (sorted for consistency) …
         for column in sort_by_name(by_type[SelectColumn]):
             table = get_root_frame(column.source)
+            # We're only interested in "standard" tables here
```

Comment:
are we, though? Does this mean that if a user defines a dataset drawing from both one or more "standard" and one or more Inline tables, the fields from the latter will be omitted from the dummy data?